### DISCUSSION TITLE
💡 Add close button to the video player

### DISCUSSION THREAD

**Discussion by syfq91** [DonutWare/Fladder#88] [Ideas]
### Summary

Add close button to quickly stop and close current video. 

![Image](https://github.com/user-attachments/assets/f52f7744-26ba-4521-ac60-7fd5d11e01f0)

### Basic Example

My first instinct when closing the video is by pressing the top left arrow button, but it will just minimise the video (which it’s working as intended). It requires another click on the stop button on the mini player to close the video.
Besides, the stop button is hidden in the minimise player when viewing from portrait orientation.

![Image](https://github.com/user-attachments/assets/6ffcee2b-8e20-4dfe-ad18-45a359faf006)

It’s not really obvious to me that I can just click stop button in the video player to close it. 

Adding the close(x) button in my opinion will make it more obvious to stop the video. 

### Drawbacks

The stop button becomes redundant. 
Maybe remove the stop button if you decide to add the close button?

**Comment by PartyDonut** [comment]
Yeah good idea the close/stop button is also just a rounded square at this point might not be entirely clear what it does.

You can stop/close the video by swiping the mini-player down. That's something that needs to be more clear as wel.
